projects/samples/devices/worlds/.console.wbproj

```diff
@@ -0,0 +1,10 @@
+Webots Project File version R2020a
```

Comment:
Is it okay to have a big console output window ? Or do we want to see the code part a little more ?

Reply:
On my small screen the problem is more the width of the text editor, the 3D view is so small that I don't see the play etc. buttons.